packages/ton-client/src/shaders/sdf.glsl

```diff
@@ -40,18 +40,18 @@ float sdTriPrism( vec3 p, vec2 h )
   return max(q.z-h.y,max(q.x*0.866025+p.y*0.5,-p.y)-h.x*0.5);
 }
 
-float sdf(in vec3 p) { 
-  //return sdTriPrism(p, vec2(1, 1)); 
-  //return length(p - vec3(0.5,0,0)) - 0.5; 
+float sdf(in vec3 p, in vec3 scale) {
+  //return sdTriPrism(p, vec2(1, 1));
+  //return length(p - vec3(0.5,0,0)) - 0.5;
   p = rotateY(p, p.y*1.);
-  return sdBox(p, vec3(0.75)) - 0.1; 
```

Comment:
ooof, I just realized another comment wasn't saved. 

I said here, that you should have a look at line 11 at the function `diamonds` and how I do a scaling there. 

If you want scale to apply to all the sdf, I think you can divide `vec3 p` by `scale` before using it and multiply with `scale` afterwards. 

I know, it's just a test, but if you use it as the second parameter of sdBox in the end, you should call it `boxSize` throughout. If you call it scale, make it behave like scale.

Reply:
I will have a look tomorrow, thanks